jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/classes/ClassesLoadedHandler.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jfr.metrics.internal.classes;
+
+import static io.opentelemetry.contrib.jfr.metrics.internal.Constants.UNIT_CLASSES;
+import static io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler.defaultMeter;
+
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler;
+import java.time.Duration;
+import java.util.Optional;
+import jdk.jfr.consumer.RecordedEvent;
+
+public final class ClassesLoadedHandler implements RecordedEventHandler {
+  private static final String METRIC_NAME_LOADED = "process.runtime.jvm.classes.loaded";
+  private static final String METRIC_NAME_UNLOADED = "process.runtime.jvm.classes.unloaded";
+  private static final String METRIC_NAME_CURRENT = "process.runtime.jvm.classes.current_loaded";
```

Comment:
Do we document somewhere what the difference is between classes.loaded and classes.current_loaded? Maybe link to that definition here?

Reply:
I added some comments for further clarity. The metric descriptions also add some information.